app/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,6 @@
     <color name="colorPrimary">#3F51B5</color>
     <color name="colorPrimaryDark">#303F9F</color>
     <color name="colorAccent">#FF4081</color>
+
+    <color name="droidcon_green">#8ACE01</color>
```

Comment:
Albo uzywajac nazwy koloru wg http://chir.ag/projects/name-that-color/#8ACE01
to nazwa moze byc "Pistachio"